### PR TITLE
[v6r13] FIX exception in the RMS with some RegisterFile Operation

### PR DESCRIPTION
**Pull request by chaen**
Some argument could be None, so split wouldn't work

### CODE REVIEW

**Comment by phicharp** [comment]
Review OK

File: RequestManagementSystem/Client/Operation.py [hunk 1]
```diff
@@ -201,12 +201,12 @@ def __len__( self ):
   @property
   def sourceSEList( self ):
     """ helper property returning source SEs as a list"""
-    return self.SourceSE.split( "," )
+    return self.SourceSE.split( "," ) if self.SourceSE else ['']
```
Comment (andresailer) [inline RequestManagementSystem/Client/Operation.py]: Are you intentionally putting an empty string in the list?
[''] does not evaluate to false.

Comment (chaen) [inline RequestManagementSystem/Client/Operation.py]: I know, this was the old behavior. There are places where we do Operation.targetSEList[0], which certainly not the right thing to do, but this requires much more digging that can't be done now. However this fix is necessary for the RMS to execute some operation, it is hotfixed

Comment (andresailer) [inline RequestManagementSystem/Client/Operation.py]: Ok, I understand.

**Pull request merged**